sources/nodes/kube.go

```diff
@@ -78,8 +78,15 @@ func (self *kubeNodes) List() (*NodeList, error) {
 			nodeList.Items[Host(node.Name)] = Info{PublicIP: node.Status.HostIP, InternalIP: addrs[0].String()}
 			goodNodes = append(goodNodes, node.Name)
 		} else {
-			glog.Errorf("Skipping host %s since looking up its IP failed - %s", node.Name, err)
-			self.recordNodeError(node.Name)
+			ip := net.ParseIP(node.Name)
```

Comment:
Ahh. Thanks for explaining. +1 to splitting the PRs and thanks for identifying and fixing the namespace handling bug :+1: 
Let us know if you are not able to find a way to get around the hostname resolution issue and I can explore to see if the proxy server on the kube-master can proxy traffic to kubelets instead of having to talk to the kubelets directly.

What does `no.Status.HostIP` point to in the case of Rackspace? Does it point to an internally accessible IP with the kubelet port exposed?